broker/plugins/kubejobs/plugin.py

```diff
@@ -232,6 +251,17 @@ def errors(self):
         except redis.exceptions.ConnectionError:
             return ()
         return self.rds.lrange("job:errors", 0, -1)
+    
+    @set_timeout(1.0)
+    def persist_state(self):
+        atual_representation = self.__repr__()
```

Comment:
```suggestion
        new_representation = self.__repr__()
```

Reply:
I think that "new_representation" isn't the best name, so I used "obj_representation".